.github/workflows/flow-coverage.yml

```diff
@@ -38,12 +39,23 @@ jobs:
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       run: ./deps/readies/bin/getredis --with-github-token --branch unstable
-    - name: Build and test
-      run: make coverage QUICK=1 SHOW=1 REJSON=${{ env.REJSON }} REJSON_BRANCH=${{ inputs.rejson-branch }}
-    - name: Upload coverage
+    - name: Build and test flow
+      run: make coverage-flow QUICK=1 SHOW=1 REJSON=${{ env.REJSON }} REJSON_BRANCH=${{ inputs.rejson-branch }}
```

Comment:
Consider setting the variables via the `env` as we do in `task-test.yml`

Reply:
Very soon in #5909